Cargo.toml

```diff
@@ -1,7 +1,8 @@
 [workspace]
 members = [
   "lib-didethresolver",
```

Comment:
would 'didethresolver' be the name of what is being published?   I'm not sure we need an extension to both packages.    Would be nice if our crate is called didethresolver then the binary can have a local name

Reply:
the name of the crates being published is in `Cargo.toml` of each folder, we can change these to be local `lib` and `gateway`, and the cratenames be `lib-didethresolver` and `didethresolver-gateway`. I would prefer it this way, actually

Reply:
moved to shorter names in latest commit